docker/r/Dockerfile

```diff
@@ -98,18 +98,18 @@ USER ${NB_USER}
 
 # set up conda in the NB_USER environment
 RUN echo ". ${CONDA_PATH}/etc/profile.d/conda.sh" >> ~/.bashrc && \
-    echo "conda activate base" >> ~/.bashrc
+    echo "conda activate base" >> ~/.bashrc && \
+    cat /renku/bashrc >> $HOME/.bashrc && \
+    echo "source ~/.bashrc" >> ~/.bash_profile
```

Comment:
I think we need to do this in the main `py` image as well so that it gets applied to all other images?

Or is there a specific reason that this is only done in the R image but not anywhere else?

Reply:
I am referring to `echo "source ~/.bashrc" >> ~/.bash_profile`

Reply:
tbh I wasn't sure why the bash_profile was used in the R image - I presume it has something to do with the specifics of how R environments are set up. I just copied that line from another place in the Dockerfile, I didn't add it in this PR.